app/build.gradle

```diff
@@ -44,15 +44,14 @@ dependencies {
     implementation "com.github.devlight.navigationtabstrip:navigationtabstrip:$rootProject.navTabStripVersion"
     implementation "com.afollestad.material-dialogs:core:$rootProject.materialDialogsVersion"
     implementation "com.jakewharton:butterknife:$rootProject.butterKnifeVersion"
-    implementation 'com.android.support:support-v4:27.1.1'
+    implementation "com.android.support:support-v4:$rootProject.supportLibraryVersion"
     annotationProcessor "com.jakewharton:butterknife-compiler:$rootProject.butterKnifeVersion"
-    implementation 'com.github.medyo:android-about-page:1.2.4'
-    implementation 'com.github.tiagohm.MarkdownView:library:0.19.0'
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.4'
-    releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.4'
-    testImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.4'
-
-    compile 'com.github.mirrajabi:search-dialog:1.1'
+    implementation "com.github.medyo:android-about-page:$rootProject.androidAboutPageVersion"
+    implementation "com.github.tiagohm.MarkdownView:library:$rootProject.markDownViewVersion"
+    implementation "com.github.mirrajabi:search-dialog:$rootProject.searchDialogVersion"
```

Comment:
I think this will use Search Dialog Version 1.2.1 and so it will create issues while again implementing search bar as it is the latest version with some changes that needs to be defined for the search bar to work. So, I suggest you to keep hardcoded version